index.php

```diff
@@ -4,6 +4,74 @@
 <head>
   <title>13th Annual Alumni Meet &middot; Home</title>
   <?php include 'topHead.php'; ?>
+ <style>
+ .logo{margin:15px;}
+ <!--#myCarousel{display:box;width:500px;height:175px;;margin-top:-75px;margin-left:-40px;}-->
+ .item{margin-left:155px;}
+ 
+.carousel-fade {
```

Comment:
@punit121 

I am not sure what this CSS is doing here. Please move it to a new file `css/carousel.css` and include that file here.